package.json

```diff
@@ -23,7 +23,8 @@
   },
   "scripts": {
     "start": "concurrently --kill-others-on-fail --names static,admin \"yarn start:static\" \"yarn start:admin\"",
+    "start:dev": "cross-env GIT_REPO_DIRECTORY=exampleSite concurrently --kill-others-on-fail --names static,admin \"yarn start:static\" \"yarn start:admin\"",
     "start:static": "hugo server -s exampleSite/ --themesDir=../.. --disableFastRender",
```

Comment:
This bit looks for `exampleSite/` which wouldn't exist outside of the theme.

Reply:
The `GIT_REPO_DIRECTORY` environment variable is propagated down to the `start:static` invocation, which then uses that variable (per the [docs](https://github.com/netlify/netlify-cms/tree/master/packages/netlify-cms-proxy-server#custom-configuration))  - at least it did on OSX; this seemed cleaner than duplicating both `start:static` and `start:admin` scripts

Reply:
I think `GIT_REPO_DIRECTORY` only applies to netlify, hugo itself is being given `-s exampleSite/` which does not exist outside of the theme directory. Here's the error I get (I'm on Linux) when I try to run `start` from within `balance-static`:

![image](https://user-images.githubusercontent.com/65056303/192408956-f85a23a0-d55e-4e23-8d76-1e8b96bda483.png)

Reply:
Are you running this as part of balance-static, or part of glam? I guess I can hook it up in a test balance-static branch or something.

Reply:
As part of `balance-static`, `start:dev` looks like it works fine.

Reply:
check again with the latest update; no errors on my side(but no guarantee the admin works because it doesn't have the dynamic admin changes or dev.yaml yet)